x-pack/plugins/infra/public/components/subscription_splash_content.tsx

```diff
@@ -103,16 +102,62 @@ export const SubscriptionSplashPrompt: React.FC = () => {
     );
   }
 
+  return (
+    <PageTemplate {...templateProps} isEmptyState={true}>
+      <EuiEmptyPrompt
+        iconType={'visLine'}
+        title={<h2>{title}</h2>}
+        body={
+          <EuiText>
+            <p>{description}</p>
+          </EuiText>
+        }
+        actions={cta}
+      />
+    </PageTemplate>
+  );
+};
+
+export const SubscriptionSplashPrompt: React.FC = () => {
+  const { licenseManagement, http } = useKibanaContextForPlugin().services;
+  const licensePageUrl = http.basePath.prepend(LICENSE_PAGE_PATH);
+  const manageLicenseURL = licenseManagement?.locator
+    ? licenseManagement?.locator?.useUrl({
+        page: 'dashboard',
+      })
+    : licensePageUrl;
```

Comment:
This logic is already being used by the component above. Do you think we'll be using it elsewhere? Maybe you could create a custom hook that returns the URL?